README.md

```diff
@@ -36,35 +41,44 @@ docker build -f Dockerfile-zeth -t zeth-dev .
 # Start the zeth development container
 docker run -ti -p 50051:50051 --name zeth zeth-dev:latest
 
-## All the commands below are ran in the docker container
+# All the commands below are run in the docker container
 # Configure your environment
 . ./setup_env.sh
 
 # Compile the proving server
 mkdir build
 cd build
 cmake .. [<flags (see below)>]
-## (optional) Run the tests
-make check # Builds and run the tests (once the tests are built, calling "make test" suffices to execute them)
-## Compile
+# Compile all libraries and tools, including the prover_server
 make
+# (optional) Run the unit tests
+make test
+# (optional) Run the all tests (unit tests, syntax checks, etc)
+make check
 
-# Start the proving server
-prover_server
+# Start the prover_server process
+src/prover_server
```

Comment:
why `src/prover_server`? The path to the binaries is set by `. ./setup_env.sh`

Reply:
reverted